tests/check/signed_video_helpers.c

```diff
@@ -134,6 +134,7 @@ create_signed_nalus_with_sv(signed_video_t *sv, const char *str, bool split_nalu
     } else {
       rc = signed_video_add_nalu_part_for_signing_with_timestamp(
           sv, item->data, item->data_size, &g_testTimestamp, true);
+      ck_assert_int_eq(signed_video_is_golden_sei(sv, item->data, item->data_size), false);
```

Comment:
Since signed_video_is_golden_sei() returns a boolean I think it is better to use ck_assert().

Also, add a comment why this is test is applied here and not if the NALU is split.